compose/ui/ui/src/jsMain/kotlin/androidx/compose/ui/Actuals.js.kt

```diff
@@ -16,8 +16,14 @@
 
 package androidx.compose.ui
 
+import kotlinx.browser.window
+
 internal actual fun areObjectsOfSameType(a: Any, b: Any): Boolean {
     return (js(
         "Object.getPrototypeOf(a).constructor == Object.getPrototypeOf(b).constructor"
     )) as Boolean
 }
+
+internal actual fun currentTimeMillis(): Long {
+    return window.performance.now().toLong()
```

Comment:
Does a raw `Date` object allow doing this without relying on `window` object?

Reply:
Date.now() can be used for actual system date/time, but it can be changed by a user in system settings.

window.performance.now() is a monotonic time, not affected by system settings.